<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="pokemonName"/>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QListWidget" name="moveList"/>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="pokemonList"/>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="gen1">
         <property name="text">
          <string>Gen 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gen2">
         <property name="text">
          <string>Gen 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gen3">
         <property name="text">
          <string>Gen 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gen4">
         <property name="text">
          <string>Gen 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gen5">
         <property name="text">
          <string>Gen 5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTabWidget" name="pokeMoves">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Level Moves</string>
       </attribute>
       <widget class="QListWidget" name="levelMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tutor Moves</string>
       </attribute>
       <widget class="QListWidget" name="tutorMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Egg Moves</string>
       </attribute>
       <widget class="QListWidget" name="eggMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Special</string>
       </attribute>
       <widget class="QListWidget" name="specialMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>TM &amp;&amp; HM</string>
       </attribute>
       <widget class="QListWidget" name="tmMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Pre Evo</string>
       </attribute>
       <widget class="QListWidget" name="preMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Dream World</string>
       </attribute>
       <widget class="QListWidget" name="dwMoves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="save"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCtrl_S">
   <property name="text">
    <string>Ctrl + S</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
